spring-integration-core/src/main/java/org/springframework/integration/store/AbstractKeyValueMessageStore.java

```diff
@@ -359,7 +358,7 @@ public void setLastReleasedSequenceNumberForGroup(Object groupId, int sequenceNu
 	}
 
 	@Override
-	public Message<?> pollMessageFromGroup(Object groupId) {
+	public Message<?> pollMessageFromGroupInner(Object groupId) {
```

Comment:
DITTO about all the new method with `do..` instead, plus their `public` must be `protected`.